app/src/lib/ci-checks/ci-checks.ts

```diff
@@ -610,6 +611,16 @@ export function getCheckRunsGroupedByActionWorkflowNameAndEvent(
     groups.set(group, newGroup)
   }
 
+  const sortedGroupNames = getCheckRunGroupNames(groups)
+
+  sortedGroupNames.forEach(gn => {
+    const group = groups.get(gn)
+    if (group) {
```

Comment:
Nit: explicit check?
```suggestion
    if (group !== undefined) {
```